### ISSUE TITLE
Support mydumper 

### ISSUE THREAD

**Issue by shenli** [pingcap/tidb#1690]
We need to support some SQL statements to support mydumper.

**Comment by tiancaiamao** [comment]
I'm looking into it and find those problem so far.

TiDB doesn't support those commands:

```
SHOW PROCESSLIST
FLUSH TABLES WITH READ LOCK
```

/*!40108 WITH CONSISTENT SNAPSHOT */ don't parsed as comment and skiped:

```
START TRANSACTION /*!40108 WITH CONSISTENT SNAPSHOT */
```

**Comment by coocood** [comment]
Fixed by #1701 & #1703
